seqr_scripts/seqr_sample_qc.py

```diff
@@ -0,0 +1,530 @@
+import logging
+import argparse
+import pickle
+
+import hail as hl
+import hail.expr.aggregators as agg
+from gnomad.sample_qc.pipeline import get_qc_mt
```

Comment:
small thing, should there be a return between the hail and gnomad imports?

Reply:
I think because gnomad is a package/third party import it gets grouped with hail